infra/scripts/docker/buildRunDocker.sc

```diff
@@ -0,0 +1,141 @@
+
+import $file.^.common.display
+import $file.^.common.build
+import $file.vars
+
+import build._
+import display._
+import vars._
+import ammonite.ops._
+import ammonite.ops.ImplicitWd._
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, Future, Promise}
+import scala.concurrent.ExecutionContext.Implicits.global
+import java.util.{Timer, TimerTask}
+import scala.util.Try
+
+def performSetup(implicit stackType: StackType): Unit = {
+  implicit val progressBar = ProgressBar(System.out, "START", "Starting setup...")
+
+  progressBar.start()
+  progressBar.stepInto("Run docker images...")
+
+  val appsInParadigm = apps.map { a => s"${a._1}-${stackType.paradigm}" -> a._2 } ++ backingServices
+
+  // Tests and publishes sync/async services to local docker registry
+  buildPublishApps(appsInParadigm.map(_._1))
+
+  // Sets up networking shared among containers
+  val networkName = setupNetworking()
+
+  // Runs cassandra container
+  runCassandra(networkName)
+
+  // Runs apps docker images
+  val startedApps = runDockerImages(appsInParadigm, networkName)
+
+  Await.ready(
+    startedApps, Duration.Inf
+  )
+
+  progressBar.finished()
+}
+
+private def buildPublishApps(apps: Seq[String])(implicit progressBar: ProgressBar, stackType: StackType): Unit = {
+  buildStack(apps, localRepo = true, test = false)
+}
+
+private def runCassandra(networkName: String)
+                           (implicit progressBar: ProgressBar): Unit = {
+  progressBar.stepInto("Cassandra")
+  progressBar.show("Start up...")
+
+  Future {
```

Comment:
we don't want to see this logs in common app logging - cassandra just spits too much of it

And yeah, this is definitely not perfect as if cassandra fails, we won't know about it. I sort of cut the corners here.